<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParticleTrackerUI</class>
 <widget class="QWidget" name="ParticleTrackerUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Particle Tracker</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="mainTabWidget">
     <property name="maximumSize">
      <size>
       <width>360</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Select Particles</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Species:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="speciesSelector_comboBox"/>
       </item>
       <item>
        <widget class="QWidget" name="selectorPlot_widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>360</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>360</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="selectorPlot_layout"/>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSlider" name="selectorTimeStep_Slider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="selectorTimeStep_lineEdit">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Particles in selected area:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="selectAll_checkBox">
         <property name="text">
          <string>Select all</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="particleList_listWidget"/>
       </item>
       <item>
        <widget class="QPushButton" name="trackParticles_Button">
         <property name="text">
          <string>Track Selected Particles</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Plot</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mainPlot_widget" native="true">
     <layout class="QVBoxLayout" name="mainPlot_layout"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
